borderland-tool/borderland_tool/lottery/__init__.py

```diff
@@ -150,7 +150,7 @@ def save_csv(self):
             if num_csv_registrations > 0:
                 fieldnames = self.registrations_csv[0].keys()
                 print("found {} registered users".format(
-                    num_csv_registrations-1))
```

Comment:
the -1 is important, because the csv contains headers; that's the line we should remove when counting